android/logto-client/src/main/kotlin/io/logto/sdk/android/LogtoClient.kt

```diff
@@ -139,6 +141,27 @@ open class LogtoClient(
         }
     }
 
+    fun fetchUserInfo(callback: RetrieveCallback<UserInfoResponse>) {
+        getOidcConfig { throwable, result ->
+            throwable?.let {
+                callback.onResult(it, null)
+                return@getOidcConfig
+            }
+            requireNotNull(result).let { oidcConfig ->
+                getAccessToken { throwable: Throwable?, result: AccessToken? ->
+                    throwable?.let {
+                        callback.onResult(it, null)
+                        return@getAccessToken
+                    }
+                    Core.fetchUserInfo(
+                        oidcConfig.userinfoEndpoint,
+                        requireNotNull(result).token
+                    ) { throwable, response -> callback.onResult(throwable, response) }
```

Comment:
can we directly assign `callback.onResult` to `fetchUserInfo` here

Reply:
updated